tools/fast/process.go

```diff
@@ -199,6 +200,16 @@ func (f *Filecoin) LastCmdStdErr() io.ReadCloser {
 	return f.lastCmdOutput.Stderr()
 }
 
+// LastCmdStdErrStr is a shortcut to just get the output as string
+func (f *Filecoin) LastCmdStdErrStr() string {
```

Comment:
Blocking: This is nice to have, but let's return an error here for the case that reading fails.

Reply:
meh, I was hoping to leave it since returning the error string will cause a test failure, and the dev will see immediately why, however it does break patterns...